database/reset_schema.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS trains;
```

Comment:
is there a reason you have a `reset_schema.sql` and a `schema.sql`? Could you place the `DROP TABLE` in the `schema.sql` file?